feed.php

```diff
@@ -232,7 +232,7 @@ function rss_buildItems(&$rss, &$data, $opt)
             // add date
             if (isset($ditem['date'])) {
                 $date = $ditem['date'];
-            } elseif ($ditem['media']) {
+            } elseif (isset($ditem['media'])) {
```

Comment:
If `$ditem['media'] = false`, then `isset($ditem['media'])` results in true. So I think this change will change the logic.

Reply:
Should be fixed now.